maintainers/scripts/lib_bash/temp

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This probably shouldn't have a shebang if it's only meant to be sourced.

If it should have a shebang, probably should have it match the one in nix-diff.sh at least.